src/ParallelAlgorithms/ApparentHorizonFinder/Criteria/Residual.cpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+#include <pup.h>
+
+#include "ParallelAlgorithms/ApparentHorizonFinder/Criteria/Residual.hpp"
```

Comment:
These should be switched. Our convention in `src/` cpp files is hpp first, then standard library, then spectre includes.

Reply:
🤦‍♂️ I knew that, just missed it somehow. Thanks for catching this